mmv1/third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -2858,6 +2868,27 @@ func resourceContainerClusterUpdate(d *schema.ResourceData, meta interface{}) er
 		log.Printf("[INFO] GKE cluster %s Private IPv6 Google Access has been updated", d.Id())
 	}
 
+	if d.HasChange("ip_allocation_policy") {
+		// Stack type change is not supported in GKE at the moment
```

Comment:
Does this mean that `stack_type` is immutable after creation? Then, in Terraform, we need to add `forcenew` to the field to make sure that when users can the value of the field in their configuration, the resource will be deleted and recreated.

Reply:
Yes, changing the stack_type is not supported in the current GKE releases. I'll label it with "forceNew". Thanks!

Error: googleapi: Error 400: Updating stack_type is not supported.
│ Details:
│ [
│   {
│     "@type": "type.googleapis.com/google.rpc.DebugInfo",
│     "detail": "[ORIGINAL ERROR] cloud-kubernetes::3: Updating stack_type is not supported. [google.rpc.error_details_ext] { code: 3 message: \"Updating stack_type is not supported.\" }"
│   }
│ ]
│ , badRequest